docs/design/epm/design.md

```diff
@@ -0,0 +1,206 @@
+# Overview
+
+This document introduces EPM fundamental and design. EPM is a module optimized enclave boot time. It is designed as two parts, one is as OCI bundles cache management, the other one is as enclave cache pool management. This documentation will focus on enclave cache pool management currently, briefly call it as EPM as below. 
+
+The EPM is used to cache enclaves which can store various enclave runtime, e.g. skeleton, Graphene (under plan), Occlum(under development). Current implementation is based on skeleton using sgx1 not supporting to extend extra pages once the enclave initialization is done. Inclavare-containers provides skeleton as an enclave runtime reference. All the introduction to EPM is based on skeleton.
```

Comment:
Add the URL of `skeleton`、`Graphene`, and `Occlum` when they firstly appear.

Reply:
using `sgx1` or `sgx2`?

Reply:
sgx1